accname/name/comp_labelledby.html

```diff
@@ -13,14 +13,36 @@
 
 <p>Tests the <a href="https://w3c.github.io/accname/#comp_labelledby">#comp_labelledby</a> portions of the AccName <em>Name Computation</em> algorithm.</p>
 
-<div role="group" aria-labelledby="h" class="ex" data-expectedlabel="div group label" data-testname="div group explicitly labelledby heading">
-  <h2 id="h">div group label</h2>
+<!--
+
+  Some overlap with tests in:
+  - /accname/name/comp_name_from_content.html
+  - /accname/name/shadowdom/*
+
+-->
+
+<div role="group" aria-labelledby="h" class="ex" data-expectedlabel="first heading" data-testname="div group explicitly labelledby heading">
+  <h2 id="h">first heading</h2>
   <p>text inside div group</p>
 </div>
 
+<div role="group" aria-label="self label" id="g2" aria-labelledby="g2 h2" class="ex" data-expectedlabel="self label + first heading" data-testname="div group explicitly labelledby self and heading">
+  <h2 id="h2">+ first heading</h2>
+  <p>text inside div group</p>
+</div>
+
+<nav aria-labelledby="s1 s2 s3 s4" class="ex" data-expectedlabel="verify spaces between foreach" data-testname="nav with verified spaces appended between each of IDREFS">
+  <span id="s1">verify</span><span id="s2">spaces</span><span>FAIL IF INCLUDED</span><span id="s3">between</span><span id="s4">foreach</span>
+</nav>
+
 <!--
 
-Todo: test all remaining cases of https://w3c.github.io/accname/#comp_labelledby
+  BLOCKED on https://github.com/w3c/accname/issues/209
+
+  Once that's resolved, it may be worthwhile to add a new ./comp_labelledby_recursion.html file,
+  so that the implementation diffs rolling in don't penalize this Interop 2024 test.
+
+  All remaining cases of https://w3c.github.io/accname/#comp_labelledby
```

Comment:
Reviewers: Some follow-on work is blocked by https://github.com/w3c/accname/issues/209 so additional tests will come separately in https://github.com/web-platform-tests/interop-2023-accessibility-testing/issues/68